### ISSUE TITLE
`.exe` suffix is used for OIDN, even on Linux

### ISSUE THREAD

**Issue by abb128** [Naxela/The_Lightmapper#47]
When attempting to use Intel's OpenImageDenoise on Linux with Release 0.3 of The Lightmapper, the following error appears in the output log:

```
Denoiser search: /home/username/Downloads/oidn-1.2.3.x86_64.linux/bin/oidnDenoise.exe
/bin/sh: 1: /home/username/Downloads/oidn-1.2.3.x86_64.linux/bin/oidnDenoise.exe: not found
```

In Linux, executable files typically don't have any extension and are simply marked as executable. This is the case for OIDN, where the executable is simply named `oidnDenoise`. Although the user could simply rename the file to `oidnDenoise.exe` for a temporary workaround, it'd be nice to properly support Linux executables.

**Comment by Naxela** [comment]
Hi and thanks for reporting the issue

I recently fixed this issue in the upcoming 0.4 release, along with some other linux specific bugs including OpenCV installation. Even though it's still in development, it's reached a point where it's fairly stable - You can try it out here: https://github.com/Naxela/The_Lightmapper/tree/0.4

Thanks again

**Comment by Naxela** [comment]
Should be fixed in the 0.4 version.
